src/Test/Integration/DbHelperAbstract.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
```

Comment:
Yes, but for new files I think it should be fine. I didn't want to introduce it for existing stuff, but for new files we should have it by default, so if anyone starts using it they will start with proper types from the ground up.